REVIEW_GUIDELINES.md

```diff
@@ -76,6 +76,29 @@ image:
 * The use of the `default` function should be avoided if possible in favor of defaults in `values.yaml`.
 * It is usually best to not specify defaults for resources and to just provide sensible values that are commented out as a recommendation, especially when resources are rather high. This makes it easier to test charts on small clusters or Minikube. Setting resources should generally be a conscious choice of the user.
 
+## Persistence
+
+* Persistence should be turned off by default
```

Comment:
Actually we prefer to have persistence via PVC enabled by default, and it is in most of the existing charts.